sdk/search/Microsoft.Azure.Search/tests/Tests/FieldBuilderTests.cs

```diff
@@ -345,8 +345,9 @@ private void OnlyTrueFor(Type modelType, Func<Field, bool> check, params string[
                 modelType,
                 fields =>
                 {
-                    foreach ((string fieldNameFromModel, Field field) in fields)
+                    foreach (string fieldNameFromModel in fields.Keys)
```

Comment:
What did the usage of ValueTuple break (I'm ok with this change, just curious)?

Reply:
The reason is that your tests build and run on .NET 4.6.1 now as well and for that target it didn't support this new deconstruct language feature so we just expanded it out.